requirements/tests.txt

```diff
@@ -1,5 +1,6 @@
-coveralls==1.8.1
-pytest==4.6.3
-pytest-cov==2.7.1
-pytest-mock==1.10.4
+coveralls==3.3.1
+pytest==7.1.3
+pytest-cov==3.0.0
+pytest-mock==3.8.2
+PyYAML==6.0
```

Comment:
Could you please explain why exactly this import is needed? None of our code uses this package and if it's coming from a dependency it should be automatically be installed.

Reply:
you're right! Didn't check the package details then, installing coveralls with extra package should resolve it. Will send a fix soon.
https://github.com/TheKevJames/coveralls-python/blob/master/setup.py#L43